commons/src/main/scala/com/lynbrookrobotics/potassium/commons/electronics/DynamicBraking.scala

```diff
@@ -0,0 +1,47 @@
+package com.lynbrookrobotics.potassium.commons.electronics
+
+import squants.QuantityRange
+import squants.electro.{ElectricPotential, Volts}
+import squants.motion.AngularVelocity
+
+object DynamicBraking {
+  private val ditherPatterns = List(
+    "............", // 0
+    "|...........", // 1
+    "|.....|.....", // 2
+    "|...|...|...", // 3
+    "|..|..|..|..", // 4
+    "|.|.|.|..|..", // 5
+    "|.|.|.|.|.|.", // 6
+    "||.||.|.|.|.", // 7
+    "||.||.||.||.", // 8
+    "|||.|||.|||.", // 9
+    "|||||.|||||.", // 10
+    "|||||||||||.", // 11
+    "||||||||||||" //  12
+  )
+    .map(_.toCharArray)
+    .map(_.map(_ == '|'))
+  private val ditherLength = ditherPatterns.map(_.length).min
+  private var tick = Byte.MinValue
```

Comment:
This may be used in different places, so instead of having a global tick you should take this as a parameter.